docs/additional-configuration.adoc

```diff
@@ -325,7 +327,9 @@ spec:
         attributes:
           container-overrides: {"resources":{"limits":{"nvidia.com/gpu":"1"}}}
 ----
-Note that the container-overrides field does not allow configuring fields that are available through the regular DevWorkspace API. In particular, workspaces will fail to start if the container-overrides attribute attempts to override `name`, `image`, `command`, `args`, `ports`, `volumeMounts`, or `env`.
+Note that the container-overrides field does not allow configuring most fields that are available through the regular DevWorkspace API. In particular, workspaces will fail to start if the container-overrides attribute attempts to override `name`, `image`, `command`, `args`, `ports`, or `env`.
```

Comment:
We could link the actual devfile Schema/API here, i.e. "Note that the container-overrides field does not allow configuring most fields that are available through the regular [DevWorkspace API](https://devfile.io/docs/2.2.2/devfile-schema#components-container)".

 Though, this is kind of awkard (since it's the DevFile API, not the DevWorkspace API...)

Also, that link will eventually become outdated, as it points to v2.2.2 of the DevFile API (and it doesn't seem there's a way to make the link point to "latest"). 

Take this suggestion with a grain of salt :P